client/src/client/__init__.py

```diff
@@ -53,19 +52,15 @@ def __init__(self) -> None:
 
         self.event_loop = asyncio.get_event_loop()
         self.db_manager = DatabaseManager(
-            str(self.startup.db_path), self.password
-        )
-        self.db_manager.initialize()
-
-        # Set identity
-        self.myself = self.db_manager.add_friend(
+            name=str(self.startup.db_path),
+            password=self.password,
             username="Alice",
-            id=digest_key(self.pub_key),
             color="blue",
-            date_added=datetime.now(),
         )
+        self.db_manager.initialize()
 
-        assert self.myself
+        # Set identity
+        assert self.db_manager.myself
```

Comment:
"myself" definitely should not be a member of DatabaseManager. This is even suggested by the name of the variable, which refers to "me, the client", not "me, the database manager"

Reply:
Naming convention could be better, I agree.